index.html

```diff
@@ -5,20 +5,52 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="/static/css/main.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
```

Comment:
Do we need this here if we import it in the scss?

Reply:
No. I moved it to sass file so it is not in html anymore.